docs/ECOS.md

```diff
@@ -0,0 +1,141 @@
+# Edge Containers
+
+## Introduction
+
+While EVE provides a lot of sophisticated device management functionality, at the end of the day its success depends on a diverse and growing ecosystem of applications that it can support. EVE is *not* a traditional operating system in that it doesn't support applications represented as executable files and [POSIX processes](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_295). Instead EVE expects its applications to be:
```

Comment:
It confusing to state this way it doesn't support POSIX processes. I would rework this part stating that EVE is remotely managed run time, allowing users to run VM and OCI containers.

Reply:
Ok, @nordmark @deitch -- I think the whole ELF/POSIX is a fair point -- I rewrote that portion. However, on the flip side, positioning EVE too close to Linux is also way too confusing. EVE is an engine -- its not even an operating system strictly speaking. Its just an engine that happens to run on bare metal.

Also, @deitch your suggestion is more suitable for the intro doc (which you're also reviewing) here in ECOs section we assume that the reader already went through the intro doc.